### PR TITLE
Initial 2018 stats post

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

**Comment by ericholscher** [comment]
@writethedocs/core Let me know if there are any other interesting stats we should be tracking. Always happy to add new bits if they are useful.

File: docs/blog/write-the-docs-2018-stats.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.. post:: Jan 7, 2018
+   :tags: stats, year in review
+   :author: Eric Holscher
+
+Write the Docs 2018 Stats
+=========================
+
+2018 was another wonderful year for Write the Docs, it marks our 6th year of existence.
+
+This post contains stats that mark our progress,
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2018-stats.rst]:
```suggestion
This post contains stats that show our progress,
```

Comment (plaindocs) [reply docs/blog/write-the-docs-2018-stats.rst]: Only because second usage in two sentences

File: docs/blog/write-the-docs-2018-stats.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+so that we can look back and see how we've changed as a community.
+
+This year I've removed a lot of the commentary except where it's useful for interpreting the numbers.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2018-stats.rst]: You're assuming people have a) read previous posts and b) care to the same level you do :-p

Comment (plaindocs) [reply docs/blog/write-the-docs-2018-stats.rst]: I'd prob just delete that sentence tbh

File: docs/blog/write-the-docs-2018-stats.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+=======
+
+* 450,000 page views (same as 2017)
+* 250,000 sessions (up from 225,000 in 2017)
+* :doc:`Guide </guide/index>`: 225,000 views
+* :doc:`Conference sites </conf/index>`: 120,000 views
+
+This is one place where we haven't seen a lot of growth.
+We haven't added much new content to the guide this past year,
+so I think that is the primary reason that traffic hasn't increased.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2018-stats.rst:73]: Are the stats for the talk video pages at all meaningful or interesting? 

I'v been wondering about timeboxing work on the docs  Guide, either myself, or all of us or with volunteers. An hour a week or something. The challenge is the org work that needs doing *before* that can meaningfully happen

File: docs/blog/write-the-docs-2018-stats.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+Thanks
+======
+
+Thanks to everyone who makes the community one of the most open and inviting in the software industry.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-2018-stats.rst]:
```suggestion
Thanks to everyone who makes this community one of the most open and inviting in the software industry.
```

File: docs/blog/write-the-docs-2018-stats.rst (hunk 2 above)
Comment (plaindocs) [reply docs/blog/write-the-docs-2018-stats.rst]: Or work it into the next in a more standalone manner.

> I'm presenting the numbers without commentary, except where it's useful for interpreting the numbers, to facilitate comparison across the years.

File: docs/blog/write-the-docs-2018-stats.rst (hunk 3 above)
Comment (plaindocs) [inline docs/blog/write-the-docs-2018-stats.rst:73]: 880 support conversations across everything

Comment (plaindocs) [reply docs/blog/write-the-docs-2018-stats.rst:73]: Could add commits and PRs as well if you're feeling fancy, but that might just be vanity metrics :-D

**Review by plaindocs — COMMENTED** [review]
Good stuff!

File: docs/blog/write-the-docs-2018-stats.rst (hunk 3 above)
Comment (plaindocs) [reply docs/blog/write-the-docs-2018-stats.rst:73]: `/videos/` the answer is not really

File: docs/blog/write-the-docs-2018-stats.rst (hunk 2 above)
Comment (thatdocslady) [reply docs/blog/write-the-docs-2018-stats.rst]: It looks like Eric is linking previous posts in the next sentence, I think it's fine to leave this. Or maybe flip them (so the links are first and then this note?)

File: docs/blog/write-the-docs-2018-stats.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+* 450 attendees in Portland (from 400 in 2017)
+* 275 attendees in Prague (from 250 in 2017)
+* 100 attendees in Australia (from 50 in 2017)
+* 50 attendees in Cincinnati (New in 2018)
+
+Australia went from a one-day to a two-day conference,
+growing from 50 to 100.
+Cincinnati was new this year,
+in collaboration with OpenHelp,
+and got 50 attendees.
```
Comment (thatdocslady) [inline docs/blog/write-the-docs-2018-stats.rst]: got > had

**Review by thatdocslady — COMMENTED** [review]
Looks fine, I second most of Sam's comments (specific replies/additions in diff).

I think that we can release this separately from the community update (which I plan to do late next week/early following), it won't hurt our traffic and there's enough content here to justify a separate post.

File: docs/blog/write-the-docs-2018-stats.rst (hunk 2 above)
Comment (mxsasha) [reply docs/blog/write-the-docs-2018-stats.rst]: > Or work it into the next in a more standalone manner.
> 
> > I'm presenting the numbers without commentary, except where it's useful for interpreting the numbers, to facilitate comparison across the years.

I like this one the most. Explaining this year's stats seems more significant than comparing this report to previous reports. And if people wonder "why is there less commentary", this explains it too.

File: docs/blog/write-the-docs-2018-stats.rst [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+Conferences
+===========
+
+* 450 attendees in Portland (from 400 in 2017)
+* 275 attendees in Prague (from 250 in 2017)
+* 100 attendees in Australia (from 50 in 2017)
+* 50 attendees in Cincinnati (New in 2018)
+
+Australia went from a one-day to a two-day conference,
+growing from 50 to 100.
```
Comment (mxsasha) [inline docs/blog/write-the-docs-2018-stats.rst]:
```suggestion
growing from 50 to 100 attendees.
```

Seems clearer?

File: docs/blog/write-the-docs-2018-stats.rst [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+.. _2017: http://www.writethedocs.org/blog/write-the-docs-2017-stats/
+.. _2016: http://www.writethedocs.org/blog/write-the-docs-2016-year-in-review/
+
+Conferences
+===========
+
+* 450 attendees in Portland (from 400 in 2017)
+* 275 attendees in Prague (from 250 in 2017)
+* 100 attendees in Australia (from 50 in 2017)
+* 50 attendees in Cincinnati (New in 2018)
```
Comment (mxsasha) [inline docs/blog/write-the-docs-2018-stats.rst]:
```suggestion
* 50 attendees in Cincinnati (new in 2018)
```

(don't see why it should be `New` ?)

**Review by mxsasha — COMMENTED** [review]
Looks great overall, left a few minor notes.

File: docs/blog/write-the-docs-2018-stats.rst (hunk 3 above)
Comment (ericholscher) [reply docs/blog/write-the-docs-2018-stats.rst:73]: Yea, we don't get a ton of traffic there. The main source of traffic is Google searches for content in the guide. We should do a better job of onboarding those folks into the community tho.

Comment (ericholscher) [reply docs/blog/write-the-docs-2018-stats.rst:73]: It could be neat to promote that though, I'll add it in.

**Review by plaindocs — APPROVED** [review]

**Comment by ericholscher** [comment]
Testing out a fancy new RSS feature from mailchimp for this. 👍

**Pull request merged**